src/status_im/common/controlled_input/utils.cljs

```diff
@@ -23,32 +24,65 @@
   [state error?]
   (assoc state :error? error?))
 
-(defn- upper-limit
+(defn upper-limit
   [state]
   (:upper-limit state))
 
+(defn lower-limit
+  [state]
+  (:lower-limit state))
```

Comment:
I honestly don't think this is a good approach, keywords are already functions, and the keyword name  is the same as the function wrapping it.

Why do we add these functions?

Reply:
The general idea behind `controlled-input/utils` was to separate all related functionality including data representation from views that use it. Of course from a view you can use just a keyword to access lower limit, but that would result in a view knowing more than I think it should)

Reply:
Well, I get your idea, and can be understandable since you are abstracting  these functions to make them reusable.

On the other hand, this approach looks more as creating an extractor in OO than doing FP.
Also we aren't adding any logic to the function and it doesn't look as idiomatic Clojure.

Actually, since there's no logic in this function, we could rewrite it as:
```clojure
(def lower-limit :lower-limit)
```

But I still feel it as non-idiomatic.

I understand that what you want is to use it like:
```clojure
(utils/lower-limit state)
```

Instead of:
```clojure
(:lower-limit state)
```

So, honestly IDK how to solve it, no need to actually change it, is up to you. 

Let's just make sure that this approach is not prefered or replicated (as long as possible) in other places, since this actually is wrapping a funciton in another function with zero extra logic, it also makes the code slightly slower.